tests/test_flows.py

```diff
@@ -277,7 +331,16 @@ def test_ap_config_bss_tear_down(self):
 
         self.check_log(self.repeater1, "agent_wlan0", r"Received credentials for ssid: Multi-AP-24G-3 .* bss_type: 2")
         self.check_log(self.repeater1, "agent_wlan2", r".* tear down radio")
-        # TODO check that conn map was updated on controller
+        conn_map = self.get_conn_map()
```

Comment:
So now I understand the conn map TODO comment, but still, we should imitate the exact steps which are tested in the testbed, so once #924 is available we should convert this to check the AP Operational BSS TLV instead of relying on the connection map.

Reply:
I disagree that we should do the exact steps of the testbed.

We should *at least* do the same as the testbed, but I think it's definitely OK if we do more.